### PR TITLE
Style single programacion

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -56,6 +56,10 @@
     .background-key-color {
       background-color:<?php echo $key_color; ?>;
     }
+
+    .copy h1, .copy h2, .copy h3, .copy h4, .copy h5, .copy h6, .copy blockquote {
```
Comment (mikeyrayvon) [inline header.php:60]: how is h1,h2,h3 etc expressed in `the_content()` ?

Comment (varffs) [inline header.php:60]: user has to use the tags in the editor [there is a dropdown for html tags]

Comment (mikeyrayvon) [inline header.php:60]: wow ive never clicked that button b4

**Comment by mikeyrayvon** [comment]
maybe the slide images should be vertically centered?

**Comment by varffs** [comment]
Yeah i dont kno. only one example of a perfectly fitting image so not sure what they want other than that. For now I happy to leave as is

**Pull request merged**